src/gsy_e/gsy_e_core/sim_results/file_export_endpoints.py

```diff
@@ -397,8 +396,6 @@ def __init__(self, area: "CoefficientArea"):
 
     @property
     def labels(self) -> List:
-        if isinstance(self._area.strategy, SCMStorageStrategy):
-            return ["slot", "charge [kWh]", "offered [kWh]", "charge [%]"]
```

Comment:
I understand the reason why these are removed, but why not converting to the new SCMStorageStrategy attributes? Or did you decide that since we will not run SCM in CLI mode, it is better to leave as is?

Reply:
> but why not converting to the new SCMStorageStrategy attributes

Which attributes, there are none to be plotted.

Reply:
There is always one attribute that can be shown: traded energy. This is also the only thing reported for the infinite bus, and I think it is important to have just for the purpose of completeness

Reply:
Hmm, this is the SCM. All energy is traded, right? So this will be only a duplicate of the input profile of the SCMStorage. Do you think this is valuable to have?

Reply:
The thing is that there is no other place where we can see it as part of the results. Maybe we can see it as part of the setup file / input profile, however I think it will be more convenient and more harmonized with the other results if we had it here as well.

Reply:
done [e0d6333](https://github.com/gridsingularity/gsy-e/pull/1718/commits/e0d633312df3c7226bc7d869d65f5ee80eedc5d1)